packages/frontend/src/App.tsx

```diff
@@ -262,6 +263,16 @@ const App = () => (
                                                                         <Space />
                                                                     </TrackPage>
                                                                 </Route>
+                                                                <Route path="/projects/:projectUuid/spaces">
+                                                                    <NavBar />
+                                                                    <TrackPage
+                                                                        name={
+                                                                            PageName.SPACE
```

Comment:
What do you think if we changed this to SPACES and the SPACES in line 260 to SPACE?
This is going to screw up a bit the analytics but makes more sense.

Reply:
yeah, makes sense